scenario/utils.ts

```diff
@@ -99,13 +99,64 @@ export async function setNextBaseFeeToZero(world: World) {
 
 // Instantly executes some actions through the governance proposal process
 // Note: `governor` must be connected to an `admin` signer
-export async function fastGovernanceExecute(governor: GovernorSimple, targets: string[], values: BigNumberish[], signatures: string[], calldatas: string[]) {
-  let tx = await (await governor.propose(targets, values, signatures, calldatas, 'FastExecuteProposal')).wait();
-  let event = tx.events.find(event => event.event === 'ProposalCreated');
-  let [proposalId] = event.args;
+export async function fastGovernanceExecute(world: World, governor: GovernorSimple, targets: string[], values: BigNumberish[], signatures: string[], calldatas: string[]) {
+  // XXX See if there is a better way to determine if GovernorSimple or Bravo should be used
```

Comment:
Feels like the right thing would be to actually make GovernorSimple admins just each be ultra-powerful voters, that way the interface is exactly wrt proposals, but maybe too late for that

Reply:
Yeah didn't have enough foresight to make that decision. A little late to do it now with the launch coming up, but definitely worth considering after mainnet launch since it will help us to simplify the code paths